core/tests/protractor_utils/forms.js

```diff
@@ -179,14 +180,16 @@ var RealEditor = function(elem) {
 };
 
 var RichTextEditor = async function(elem) {
+  var rteElements = elem.all(by.css('.protractor-test-rte'));
+  var modalDialogElements = element.all(by.css('.modal-dialog'));
+  var textComponentEditorLocator = by.css(
+    '.protractor-test-close-rich-text-component-editor');
```

Comment:
Can you incorporate the fact that this is the button to close the modal into the variable name?

Reply:
Fixed!